hypothesis-python/src/hypothesis/entry_points.py

```diff
@@ -29,7 +29,14 @@
         import importlib_metadata  # type: ignore  # mypy thinks this is a redefinition
 
     def get_entry_points():
-        yield from importlib_metadata.entry_points().get("hypothesis", [])
+        try:
+            eps = importlib_metadata.entry_points(group="hypothesis")
```

Comment:
I think you want:

`entry_points().select(group="hypothesis")`

Reply:
I'm following https://importlib-metadata.readthedocs.io/en/latest/using.html#entry-points here, which says they're equivalent 🤷‍♂️

Reply:
Ah great, my misunderstanding - I only recall reading the changelog note about this.